webapp/channels/src/components/widgets/menu/menu_group.tsx

```diff
@@ -13,7 +13,7 @@ type Props = {
 /**
  * @deprecated Use the "webapp/channels/src/components/menu" instead.
  */
-const MenuGroup = ({ divider, children } : Props) => {
+const MenuGroup = React.memo(({ divider, children }: Props) => {
```

Comment:
We normally put the `React.memo` in the `export default` line, to keep this line as clean as possible, because is normally one that is more complex.